### DISCUSSION TITLE
Brew warning reported against virtuslab/scala-cli

### DISCUSSION THREAD

**Discussion by vivekragunathan** [VirtusLab/scala-cli#312] [General]
Getting this on almost brew commands on mac (update, upgrade, cleanup, doctor)

```
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the virtuslab/scala-cli tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/virtuslab/homebrew-scala-cli/scala-cli.rb:12
```

Is this a problem that needs to be fixed? Or is there a workaround?

**Comment by lwronski** [comment]
Thanks for reporting,  

It will be fixed in the next version of scala-cli. Based on discussion in [Homebrew/discussions](https://github.com/Homebrew/discussions/discussions/2311) we remove deprecated `bottle :unneeded` - [PR](https://github.com/VirtusLab/scala-cli/pull/316).
